app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt

```diff
@@ -22,26 +24,28 @@ import java.io.IOException
 class HomeFragment : Fragment() {
 
     private lateinit var homeViewModel: HomeViewModel
-    private val myGogiyumAddress = "https://gogiyum.com/food.json"
+    private val myGogiyumAddress = "https://gogiyum.com/api/menu"
```

Comment:
We usually use static const variable for fixed URL.

You can declare this variable outside of class.
```
const val MY_GOGIYUM_URL = "https://gogiyum.com/api/menu"

class HomeFragment : Fragment() {

}
```

Reply:
I changed it